spec/playlist_spec.rb

```diff
@@ -8,6 +8,17 @@
   end
 
   it 'should render master playlist' do
+    options = { playlist: 'playlist_url', bitrate: 6400,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.